metrics/src/counter.rs

```diff
@@ -258,7 +258,7 @@ mod tests {
     fn test_metricsrate() {
         try_init_logger_at_level_info().ok();
         let _readlock = get_env_lock().read();
-        env::remove_var("SOLANA_DEFAULT_METRICS_RATE");
+        unsafe { env::remove_var("SOLANA_DEFAULT_METRICS_RATE") };
```

Comment:
this is in test code marked with `serial`, so I would hope we can just keep it as is